.gitignore

```diff
@@ -73,5 +73,9 @@ dist
 .DS_Store
 .trash
 
+# dotenv environment variables file
+.env
+.env.test
+
 packages/main/ignore
-packages/root-config/cypress.env.json
\ No newline at end of file
+packages/root-config/cypress.env.json
```

Comment:
why?

Reply:
It was there before, I thought ite was referring to a .env file, didn't understand it so I put it back and added my own .env fil

Reply:
it's your addition so it couldn't have been there before take this and the rtf file out

Reply:
Ok

Reply:
Removed now